aten/src/ATen/native/AdaptiveAveragePooling3d.cpp

```diff
@@ -63,19 +65,20 @@ void adaptive_avg_pool3d_out_frame(
                 ot * osizeH * osizeW + oh * osizeW + ow;
 
             /* compute local average: */
-            scalar_t sum = 0;
+            accscalar_t sum = accscalar_t(0);
             for (const auto it : c10::irange(kT)) {
               for (const auto ih : c10::irange(kH)) {
                 for (const auto iw : c10::irange(kW)) {
                   scalar_t val =
                       *(ip + it * istrideT + ih * istrideH + iw * istrideW);
-                  sum += val;
+                  sum += accscalar_t(val);
```

Comment:
Actually not sure this cast is necessary if it supports implicit upcasting. Probably fine, but don't want to override this in CUDA as some float type may have specific overloads for reduced precision dtypes.

If this is just to silence warnings/errors probably fine and a super minor nit.